Item/Item.module.less

```diff
@@ -156,37 +177,48 @@
 			}
 		}
 
+		.disabled({
+			color: @agate-item-disabled-color;
+			border-color: @agate-item-disabled-border-color;
+
+			.focus({
+				opacity: @agate-item-disabled-opacity;
```

Comment:
I think the name is wrong.
@agate-item-focus-disabled-opacity?

Reply:
This had to be deleted. `opacity` was moved out of `focus` mixin, so the naming is ok